nx_parallel/algorithms/shortest_paths/weighted.py

```diff
@@ -1,13 +1,12 @@
 from joblib import Parallel, delayed
 from networkx.algorithms.shortest_paths.weighted import single_source_bellman_ford_path
-
 import nx_parallel as nxp
 
 __all__ = ["all_pairs_bellman_ford_path"]
 
 
-def all_pairs_bellman_ford_path(G, weight="weight"):
-    """Compute shortest paths between all nodes in a weighted graph.
+def all_pairs_bellman_ford_path(G, weight="weight", n_jobs=-1):
+    """Parallelly computes shortest paths between all nodes in a weighted graph.
```

Comment:
You are right that we need to find better wording here.  The NetworkX wording obviously doesn't include parallel. But "parallelly" is almost never used for computing. (I could be wrong here, but I haven't seen it. The definition of the word exists, but it doesn't seem to be used in computing. Please let me know if I'm wrong with this.)
Other possible words/phrases:
```python
    """Computes in parallel shortest paths between all nodes in a weighted graph.
```
```python
    """Computes, in parallel, shortest paths between all nodes in a weighted graph.
```
```python
    """Computes shortest paths between all nodes in a weighted graph in parallel
```
```python
    """Computes shortest paths between all nodes in a weighted graph (parallel version)
```
```python
    """Parallel version of networkx.all_pairs_bellman_ford_path
```
```python
    """Parallel algorithm for shortest_paths between all nodes in a weighted graph
```
Thoughts?

Reply:
Here’s another idea for wording:
Parallel computation of …..